### PR TITLE
 Move Ballerina Build, Test and Publish into a Single Task

### PR DESCRIPTION
**Pull request by gayaldassanayake**
## Purpose

Incorporate all tests, build, and push into one task.

Related to ballerina-platform/ballerina-standard-library#1247

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: oracledb-native/build.gradle [hunk 1]
```diff
@@ -82,22 +82,21 @@ spotbugsTest {
...
    doLast {
-       if (spotbugsMain.reports.size() > 0 &&
-               spotbugsMain.reports[0].destination.exists() &&
-               spotbugsMain.reports[0].destination.text.readLines().size() > 0) {
-           spotbugsMain.reports[0].destination?.eachLine {
-               println 'Failure: ' + it
-           }
-           throw new GradleException("Spotbugs rule violations were found.");
-       }
+        if (spotbugsMain.reports.size() > 0 &&
```
Comment (BuddhiWathsala) [inline oracledb-native/build.gradle:85]: Please check the indentation and formatting.

File: oracledb-native/build.gradle [hunk 2]
```diff
@@ -82,22 +82,21 @@ spotbugsTest {
...
+                spotbugsMain.reports[0].destination.exists() &&
+                spotbugsMain.reports[0].destination.text.readLines().size() > 0) {
+            spotbugsMain.reports[0].destination?.eachLine {
+                println 'Failure: ' + it
+            }
+            throw new GradleException("Spotbugs rule violations were found.");
+        }
```
Comment (BuddhiWathsala) [inline oracledb-native/build.gradle:92]: Please check the indentation and formatting.

Comment (gayaldassanayake) [reply oracledb-native/build.gradle:92]: Fixed the indentation issue with 5107ebc6e31adf2a48af0feab1cc16c3732657a1

**Comment by BuddhiWathsala** [comment]
Title case is unnecessary in a commit message. Try to practice these commit message guidelines https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md#submit-your-contribution.

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

**Pull request merged**